docs/user-guides/phase1-mainnet.md

```diff
@@ -0,0 +1,54 @@
+---
+id: phase1-mainnet
+sidebar_label: Bitcoin Staking Phase-1 Mainnet
+hide_table_of_contents: true
+pagination_prev: null
+pagination_next: null
+custom_edit_url: null
+---
+
+# Bitcoin Staking Phase-1 Mainnet
+
+💻 Official instruction list for participanting in Bitcoin Staking Phase-1
+Mainnet.
+
+---
+
+## For Wallets <a id="wallets"></a>
+
+- [Our staking web dapp](https://github.com/babylonlabs-io/simple-staking).
+
+- [Wallet integration guide](https://github.com/babylonlabs-io/networks/blob/main/bbn-1/integration/wallet.md).
```

Comment:
can use v0.2.x after the integration pr is merged

Reply:
Would like to keep the main as v0.2.x is only related to the params.